enhancements/installer/component-selection.md

```diff
@@ -52,7 +54,6 @@ mechanism that allows components to be disabled in a first class way that the in
 
 * Admins can easily explicitly exclude specific "optional" components from their cluster, at install time.
 * Admins can enable a previously excluded optional component, at runtime.
-* Install wrappers like assisted-installer can define an install-config that excludes specific components
```

Comment:
they cannot exclude specific components, because we ditched the exclusion property.  They can include explicit caps, on top of whatever comes in their baseline set.

Reply:
i think maybe you're being too literal.

the assisted installer can define an install-config in which the net result of using that install-config will be that the installed cluster has excluded some well understood capabilities.

e.g. they can set CapSet=None, enable=x,y,z.  Which, yeah, it's not "excluding" other caps explicitly, but essentially it is excluding the rest of the caps.

and that was the point of the goal, that they could achieve that end.

Reply:
I've pushed 669cc72d -> ed81a86b to restore this line, and extend it with:

> ... (by asking for components they do want, possibly an empty set)

to make it clear that the exclusion mechanism is now "ask for what you want" and no longer "list some things you don't want".